test/specs/bar.spec.js

```diff
@@ -767,6 +767,18 @@ define(['d3', 'bar', 'barChartDataBuilder'], function(d3, chart, dataBuilder) {
                 expect(defaultYAxisLabelOffset).not.toBe(newYAxisLabelOffset);
                 expect(newYAxisLabelOffset).toBe(testYAxisLabelOffset);
             });
+
+            it('should provide locale getter and setter', () => {
+                let defaultLocale = false,
+                    testLocale = 'en-GB',
+                    newLocale;
+
+                barChart.locale(testLocale);
+                newLocale = barChart.locale();
+
+                expect(defaultLocale).not.toBe(newLocale);
+                expect(newLocale).toBe(testLocale);
+            })
         });
     });
 });
```

Comment:
I would add another test within the 'render' section, checking that a label is different when using a different locale.

Reply:
I'm having trouble getting the test to pass, a little help here would be very much appreciated.

Reply:
OK, could you upload the test in a commented state?

Reply:
Sure, coming up!

Reply:
I've added the commented test, been driving me crazy haha. I hope you can take a look. @Golodhros